File: csharp/MapleStory.OpenAPI/Src/Dto/Union/UnionArtifactEffectDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 유니온 아티팩트 효과 정보
    /// </summary>
    public class UnionArtifactEffectDTO
    {
        /// <summary>
        /// 아티팩트 효과 명
        /// </summary>
        [JsonProperty("name")]
        public string Name { get; set; }

        /// <summary>
        /// 아티팩트 효과 레벨
        /// </summary>
        [JsonProperty("level")]
        public int Level { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/tms/dto/union/UnionArtifactEffectDTO.java
package dev.spiralmoon.maplestory.api.tms.dto.union;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 神器效果資訊
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class UnionArtifactEffectDTO implements dev.spiralmoon.maplestory.api.common.dto.union.UnionArtifactEffectDTO {

    /**
     * 神器效果名稱
     */
    @JsonProperty("name")
    private String name;

    /**
     * 神器效果等級
     */
    @JsonProperty("level")
    private int level;
}
